src/main/mulmo/handler.ts

```
@@ -265,8 +265,8 @@
       pdfSlide: hasMatchingAction(["pdfSlide", "pdf"], actionNames),
       pdfHandout: hasMatchingAction(["pdfHandout", "pdf"], actionNames),
     };
-    const audioContext = enables.audio ? await audio(context, settings, graphAICallbacks) : context;
-    const imageContext = enables.image ? await images(audioContext, settings, graphAICallbacks) : audioContext;
+    const audioContext = enables.audio ? await audio(context, settings.APIKEY ?? {}, graphAICallbacks) : context;
+    const imageContext = enables.image ? await images(audioContext, settings.APIKEY ?? {}, graphAICallbacks) : audioContext;
```

ESLint

[error] Replace `·?·await·images(audioContext,·settings.APIKEY·??·{},·graphAICallbacks)` with `⏎······?·await·images(audioContext,·settings.APIKEY·??·{},·graphAICallbacks)⏎·····`

(prettier/prettier)